Source/Rhetos.Utilities/Rhetos.Utilities.csproj

```diff
@@ -4,6 +4,12 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DebugType>full</DebugType>

```

Comment:
Maybe the issue could be solved by migrating other remaining projects to portable PDB type?
I'd like to keep portable PDB if possible since it's a new recommended standard, and it should work well on .NET 4.7.2.

Reply:
The way I understand a problem is that if a project targeting full .NET framework (in my case test project) references netstandard2.0 library with portable debug info, some part of the debug stack fails to load it. Our executables target full .NET framework. I can remove this change so we can try and solve this issue as a separate task?

Reply:
Change abandoned for now.